src/Core/Checkout/Document/Renderer/CreditNoteRenderer.php

```diff
@@ -185,6 +191,28 @@ public function render(array $operations, Context $context, DocumentRendererConf
                     $config->jsonSerialize(),
                 );
 
+                // set the template renderer to be able to render template
+                $doc->setTemplateOptions([
```

Comment:
Is this needed outside of v6.7.0? If not, then I'd add it inside the if statement.